index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Microverse Porfolio Setup</title>
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body>
+   
+    <header>
+      <div class="main-header">
+          <div class="Logo">
+            <a href="#">My Logo</a>
+          </div>
+          
+            <nav>
+              <div class="bar-bar">
+                <ul>
+                  <li><a href="">Porfolio</a></li>
+                  <li><a href="">About</a></li>
+                  <li><a href="">Contact Us</a></li>
+                </ul>
+
+               
+                <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                  <i class="fa fa-bars"></i>
+                </a>
+              </div>
+            </nav>
+      </div>
+    </header>
+
+
+    <section>
+        <div class="main-section-area">
+
+          <div class="main-slide">
+            <h1>I’m Alexandra<br/>
+              Glad to see you!</h1>
```

Comment:
- [ ] [OPTIONAL] While you have the right font family and weight applied to the header title, it doesn't look right.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/164454293-88b94200-dfc5-4f10-82dc-a210fb065e84.png) | ![image](https://user-images.githubusercontent.com/62290545/164454410-c0f0690c-5eb5-49c1-a054-b4041c2126d9.png) |

- The reason for that is that you're only importing font-weight `200` from google fonts.
https://github.com/ogiogiovictor/microverse_porfolio_project/blob/abd79434833aa2b6b5a15206422cf0da99f9b814/main.css#L1
- You can select multiple font-weight values while selecting the google font to be something like the following: 
```
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;700&display=swap');
```